### ISSUE TITLE
Normal data generation kicks off many times for multi-layer meshes

### ISSUE THREAD

**Issue by kbiElude** [kbiElude/Emerald#14]
The data generation process should launch only once, after all layers have their VSAs adjusted.

Reproduces with the 'robot' scene from Suxx loaded into the COLLADA Converter tool.

**Comment by kbiElude** [comment]
No longer relevant under LW Exporter Plug-in
